From 960b9b11776ab0ac3e079ab09d94e31fbb0bf7fa Mon Sep 17 00:00:00 2001
From: James <jahess.it@gmail.com>
Date: Fri, 14 Mar 2025 10:24:56 -0600
Subject: [PATCH] Fixed Compilation Error

---
 src/admin.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/admin.c b/src/admin.c
index d007b22..05f743d 100644
--- a/src/admin.c
+++ b/src/admin.c
@@ -310,8 +310,9 @@ admin_th_listen_exit(struct sockaddr *cliaddr, int sock)
  * Thread to communicate with peer
  */
 void *
-admin_th_network_peer(void *sock)
+admin_th_network_peer(void *sock_ptr)
 {
+   int32_t sock = *(int32_t *)sock_ptr;
    int16_t  n, bytes, i, fail, quantum;
    int ret;
    socklen_t len;
@@ -910,7 +911,7 @@ admin_th_network_peer(void *sock)
  * Release resources and delete acquired terminal...
  */
 void
-admin_th_network_peer_exit(struct term_node *term_node, int sock)
+admin_th_network_peer_exit(struct term_node *term_node, int32_t sock)
 {
    dlist_t *p;
    struct interface_data *iface_data;
